hazelcast/src/main/resources/hazelcast-config-5.0.xsd

```diff
@@ -2778,6 +2789,15 @@
                         <xs:enumeration value="index"/>
                         <xs:enumeration value="intercept"/>
                         <xs:enumeration value="publish"/>
+                        <xs:enumeration value="aggregate"/>
+                        <xs:enumeration value="projection"/>
+                        <xs:enumeration value="deploy"/>
+                        <xs:enumeration value="list"/>
+                        <xs:enumeration value="suspend"/>
+                        <xs:enumeration value="resume"/>
+                        <xs:enumeration value="export"/>
+                        <xs:enumeration value="upload"/>
+                        <xs:enumeration value="write"/>
```

Comment:
What's `aggregate`, `projection`, `deploy`?
Also the whole list is different from the list in `hazelcast-config-5.0.json`. I'm not sure but I have a gut feeling they should be the same.
Also, with this number of actions, maybe we should sort them alphabetically.

Reply:
`aggregate`,`projection`, `deploy` were missing in the xsd, though they are not related to this PR. I've made xsd and json files same